app/assets/stylesheets/application.css

```diff
@@ -70,7 +70,7 @@ img {
 }
 
 .pet_img {
-  width: 70%!important;
+  width: 400px!important;
```

Comment:
I'm still playing around with design here, but this seems to make the pet#show page a bit cleaner